app/models/user.rb

```diff
@@ -1,5 +1,9 @@
 class User < ApplicationRecord
-  has_many :posts
+  has_many :posts, foreign_key: :author_id
   has_many :comments
   has_many :likes
```

Comment:
- [ ] You must specify that the foreign key is `author_id` in this case where the model is different from the column name, as Rails checks for a `user_id` column. Kindly do that.

  ```
  has_many :posts, foreign_key: :author_id
  has_many :comments, foreign_key: :author_id
  has_many :likes, foreign_key: :author_id
  ```